docs/how_to_guides/how_to_use_EDB.rst

```diff
@@ -20,3 +11,46 @@ For information on how to install and set up the EDB, see :ref:`install-edb`.
     ../examples/edb/edb_example
     ../examples/edb/simple_acid_example
     ../examples/edb/multiple_phases_example
```

Comment:
@dangunter 

These are the examples that demonstrate how to use EDB to automatically generate the IDAES configs.